cmd/endpoint.go

```diff
@@ -228,25 +229,61 @@ func localEndpointsMemUsage(endpoints EndpointList) ServerMemUsageInfo {
 	}
 }
 
+// localEndpointsNetStats - returns ServerNetStatsInfo for only the
+// local endpoints from given list of endpoints
+func localEndpointsNetStats(endpoints EndpointList) ServerNetStatsInfo {
+	var tputs []xnet.Stats
+	var oneMinTputs []xnet.Stats
+	var fiveMinTputs []xnet.Stats
+	var fifteenMinTputs []xnet.Stats
+	var addr string
+	seenEndpoints := set.NewStringSet()
+	for _, endpoint := range endpoints {
+		// Only proceed for local endpoints
+		if endpoint.IsLocal {
+			if seenEndpoints.Contains(endpoint.Host) {
+				continue
+			}
+			addr = GetLocalPeer(endpoints)
+			tput := *xnet.GlobalNetStats
```

Comment:
Not sure if you wrote this for readability but it is concise enough that we can write like and it doesn't lose any readability. 

```
tputs = append(tputs, *xnet.GlobalNetStats)
```